agent/config/runtime.go

```diff
@@ -1015,6 +1016,35 @@ type RuntimeConfig struct {
 	// flag: -rejoin
 	RejoinAfterLeave bool
 
+	// RequestLimitsMode enforces the action that will occur when RequestLimitsReadRate
+	// or RequestLimitsWriteRate is exceeded.  A value of "enforce" will block
```

Comment:
Should we also include a disabled mode?  If not, do we only turn this on if `mode` is specified?  If they have read_rate or write rate should it be ignored if mode is not specified?